includes/Dashboard.php

```diff
@@ -233,7 +233,7 @@ public function load_stories_dashboard() {
 
 		$preload_data = array_reduce(
 			$preload_paths,
-			'rest_preload_api_request',
+			'\Google\Web_Stories\rest_preload_api_request',
```

Comment:
?

Reply:
it's all explained in the PR description